app/Http/Controllers/PublicController.php

```diff
@@ -28,28 +30,16 @@ public function category()
     }
     public function testimonial()
     {
-        return view('public.testimonial');
+        $testimonials = Testimonial::where('published', 1)->get();
+        return view('public.testimonial',compact('testimonials'));
     }
     public function joblist()
     {
         return view('public.job-list');
     }
-    
-    public function jobdetails(String $id)
-    {
-        $job=JobData::with(['company'])->findOrFail($id);
-        dd($job);
-        return view('public.job-detail',compact('job'));
-    }
-
-    public function jobApply(Request $request)
+    public function jobdetails()
```

Comment:
I can't understand the reason for this change.

Reply:
You asks for the changes in testimonials??

Reply:
The code in jobdetails and in jobaApply  are removed.

Reply:
I didn't remove it .esraa remove it in the last update .and we are working on admin files not public this code was in the public as I remember and ur code was editing and show in public page and this isn't correct